keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1972,6 +2022,10 @@ exist at all must not be allowed to run. Instead, a suitable event must be
 emitted which explains the problem. Such a situation can occur as part of
 downgrade scenarios.
 
+In addition, kubelet informs about the claim status with a
```

Comment:
so this condition would be added whether the pod was scheduled by kube-scheduler or not. Correct?